crates/precompile/src/bn128.rs

```diff
@@ -4,11 +4,11 @@ use crate::{
 };
 use std::vec::Vec;
 
-use substrate::{
+use arkworks::{
     encode_g1_point, g1_point_add, g1_point_mul, pairing_check, read_g1_point, read_g2_point,
     read_scalar,
 };
-mod substrate;
+mod arkworks;
```

Comment:
This was mainly a quick hacky way to enable arkworks -- alternative would have been to branch off #2266 , which would have been overkill to just test CI performance